packages/sources/icap/docs/known-issues.md

```diff
@@ -17,3 +17,7 @@ wscat -c 'ws://json.mktdata.portal.apac.parametasolutions.com:12000'
 ```
 
 - If credentials work for a single connection, open a second terminal and run the same commands while the first is still running. The expected behaviour is that both terminals should fire out a massive amount of price data.
+
+### CACHE_MAX_AGE interaction with Heartbeat messages
+
+If `CACHE_MAX_AGE` is set below a current heartbeat interval (60000ms), the extended cache TTL feature for out-of-market-hours that relies on heartbeats will not work.
```

Comment:
Could we catch this at runtime by throwing a error/warning if `CACHE_MAX_AGE` is < heartbeat?

Reply:
I had this idea to have a check when EA bootstraps. My concern was that if somehow heartbeat interval from DP side gets changed, we would need to change the code and release a new version.

Reply:
Ahh that's a good call out - one option could be to determine if we are in this bad state based on the times we receive the heartbeats, but I think that could be a potential future improvement rather than blocking this PR.